src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -69,6 +69,9 @@ module sonic-acl {
 					type stypes:packet_action;
 				}
 
+				/* Validating 'PACKET_ACTION' exist if ACL type is 'CTRLPLANE' */
```

Comment:
@bingwang-ms Is PACKET_ACTION mandatory for data plane ACL?

Reply:
No, `PACKET_ACTION` is not mandatory for data plane ACL. 
For example, we can have below config for an ACL rule
```
# redis-cli -n 4 hgetall "ACL_RULE|EVERFLOW|RULE_3"             
1) "PRIORITY"
2) "9997"
3) "MIRROR_INGRESS_ACTION"
4) "session_1"
5) "L4_SRC_PORT"
6) "4661"
```